Sources/PrimerSDK/Classes/Core/PrimerHeadlessUniversalCheckout/Models/PrimerHeadlessUniversalCheckoutPaymentMethod.swift

```diff
@@ -8,15 +8,26 @@
 
 
 import Foundation
+import PassKit
 
 extension PrimerHeadlessUniversalCheckout {
     
     public class PaymentMethod: NSObject {
         
         static var availablePaymentMethods: [PrimerHeadlessUniversalCheckout.PaymentMethod] {
-            let availablePaymentMethods = PrimerAPIConfiguration.paymentMethodConfigs?
+            var availablePaymentMethods = PrimerAPIConfiguration.paymentMethodConfigs?
                 .compactMap({ $0.type })
                 .compactMap({ PrimerHeadlessUniversalCheckout.PaymentMethod(paymentMethodType: $0) })
+            
+            if !PrimerSettings.current.paymentMethodOptions.applePayOptions!.showApplePayForUnsupportedDevice {
```

Comment:
Are we confident that this force unwrap is safe? It might be better to use ? here